packages/spacebars/package.js

```diff
@@ -1,7 +1,7 @@
 Package.describe({
   name: 'spacebars',
   summary: "Handlebars-like template language for Meteor",
-  version: '1.1.0',
+  version: '1.2.0',
```

Comment:
Why this should be a minor if the others are patch?

Reply:
Fixing